content/source/docs/cloud/integrations/service-now/index.html.md

```diff
@@ -190,3 +194,84 @@ Access Control Roles | Description
 
 For users who only need to order from the Terraform Catalog, we recommend creating another role with read-only permissions for `x_terraform_vcs_repositories` to view the available repositories for ordering infrastructure.
 
+## Example Customizations
+
+### Creating a Catalog Item with Pinned Variables
+
+This example use case is used to demo a potential solution to creating a Terraform Catalog Item for resources that limits user input to certain variables.
```

Comment:
```suggestion
This example use case creates a Terraform catalog item for resources that limits user input to certain variables.
```

I'm seeing a few instances through here that look like they shouldn't be capitalized. Let me know if these are brand names, or should be a proper noun for some other reason!